secant/Util/WalletStorage.swift

```diff
@@ -127,11 +127,9 @@ struct WalletStorage {
     func nukeWallet() {
         deleteData(forKey: Constants.zcashStoredWallet)
     }
-}
-
-// MARK: - Wallet Storage Codable & Query helpers
+    
+    // MARK: - Wallet Storage Codable & Query helpers
```

Comment:
I think a tab just sneaked in

Reply:
This comment is now inside the struct, separating some helper functions, therefore indented

Reply:
solved